### ISSUE TITLE
Visual Studio 2015 and 2017 libprotobuf for debug is missing (e. g. csgo)

### ISSUE THREAD

**Issue by thewavelength** [alliedmodders/hl2sdk#36]
See branch on csgo @ lib\win32\debug and compare to lib\win32\release

**Comment by psychonic** [comment]
This is semi-intentional.

The debug libraries are up to twice the size of the release ones and not required for a regular build. Neither are handled that well by git.

Typically, if someone is capable of compiling and debugging, they are also able to compile their own local copy of the protobuf library as well.

(That's not to say that we won't add them, and I'm not going to close this out, but that was my own reasoning for not adding them. Not sure how the other team members feel.)

**Comment by thewavelength** [comment]
Thanks, the explanation makes definitely sense.
I know Git is not intended for binary files and I already wondered that the release-versions of the library are included.

In the meantime I'll compile the lib on my own and will wait what the other team members will say.

If I can help here in any way (e. g. compile/upload VS2017 version for you), let me know :-)

**Comment by KyleSanderson** [comment]
We could use https://git-lfs.github.com/ to host these bins; maybe.

**Comment by ghost** [comment]
> We could use https://git-lfs.github.com/ to host these bins; maybe.

Yeah, there's really no reason to not have them because of git lfs.

**Comment by psychonic** [comment]
Adding via Git LFS sounds fine to me.
